### ISSUE TITLE
Windows10: bug: error when I start trsync. No syncro done/start

### ISSUE THREAD

**Issue by PhilippeAccorsi** [buxx/trsync#53]
I can seen this in log window:

~~~
[2022-06-18T22:46:24Z INFO  trsync_manager_systray] Start manager
[2022-06-18T22:46:24Z INFO  trsync_manager::security] Get password for service 'trsync::demo.tracim.fr' and user 'xandercagexxx'
[2022-06-18T22:46:24Z INFO  trsync_manager::reload] Track config file changes with C:\Users\xandercagexxx\AppData\Local\trsync.conf.track
[2022-06-18T22:46:24Z INFO  trsync_manager_systray] Raw password disabled, prepare to start password receiver
[2022-06-18T22:46:24Z INFO  trsync_manager_systray] Password receiver started on port: 5000
[2022-06-18T22:46:24Z INFO  trsync_manager_systray] Start systray
[2022-06-18T22:46:25Z INFO  trsync_manager::daemon] '12' process to start
[2022-06-18T22:46:25Z INFO  trsync_manager::daemon] '0' process to stop
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 3, kind: NotFound, message: "Le chemin d’accès spécifié est introuvable." }', manager\src\daemon.rs:145:10
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
~~~

I dont know why ``Le chemin d’accès spécifié est introuvable` is it on my local computer or something else ? (I can seen correctly the space in configuration menu).

**Comment by PhilippeAccorsi** [comment]
I think something wrong with this line ``Get password for service 'trsync::demo.tracim.fr' and user 'xandercagexxx'` because this is my local username and not username of demo account.

not ?

**Comment by buxx** [comment]
@PhilippeAccorsi `Get password for service 'trsync::demo.tracim.fr' and user 'xandercagexxx' ` is correct, username must be session username to be able to retrieve the password from keyring service. The issue bug is about something else.

Is that Window or GNU/Linux version ? Do you still reproduce this bug ?

**Comment by PhilippeAccorsi** [comment]
I have the bug each time I tried to synchronise.

**Comment by buxx** [comment]
@PhilippeAccorsi Can you retry with new version ? https://tracim.bux.fr/ui/guest-download/3b7e3502-7609-4f52-b921-da847a83e654 And paste here the new error message. Maybe you can add `RUST_BACKTRACE=1` to have more infos.

**Comment by PhilippeAccorsi** [comment]
Ok. I tried this at end of week. Thanks.

**Comment by PhilippeAccorsi** [comment]
Hello @buxx 

I have tested your last version and it is working good \o/

I have done test on demo and also private Tracim instance and it is working perfectly. 

I have tested on one space where I didn't know the size of this space before starting synchronize. At the end, I have downloaded 1.2 Gio. So I'm very intersting by this 2 issues, to not start synchro if space size is to big or to limit biggest file I want to downloads: https://github.com/buxx/trsync/issues/43 and  https://github.com/buxx/trsync/issues/41

Anyway, that is a good tool ;)
